frontend/src/components/shared/Dropdown/Selectbox.test.tsx

```diff
@@ -34,14 +34,16 @@ const getProps = (props: Partial<Props> = {}): Props => ({
 })
 
 describe("Selectbox widget", () => {
-  const props = getProps()
-  const wrapper = shallow(<Selectbox {...props} />)
-
   it("renders without crashing", () => {
+    const props = getProps()
```

Comment:
nit: would probably be a bit cleaner to declare these as undefined variables at the top of the `describe` block and instantiate them before each test in a `beforeEach`

There are a few places we'd then have to reinstantiate `props`/`wrapper` within a test since we need to use non-default props, but overall I think it's worth the number of redundant lines of code that get removed

Reply:
Oh duh, yeah, let's set defaults in a `beforeEach`.

Reply:
I just didn't like having those variables sitting in the `describe` scope, felt like an anti-pattern.